webapp/channels/src/components/add_user_to_channel_modal/add_user_to_channel_modal.tsx

```diff
@@ -258,6 +258,8 @@ export default class AddUserToChannelModal extends React.PureComponent<Props, St
 
         const content = (
             <SuggestionBox
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore
```

Comment:
This is a small hack i am not proud of, its because TS complains you cannot pass re like this to component without forwardRef. Let me know what you think. I spent too much time on this and left it like this

Reply:
I think this is actually happening because of the updated version of `@types/react-redux`. I was trying to find a better way to do this, but I wasn't able to find anything easy. You may want to roll that back, but I'm fine if this stays as-is.

To type this properly, we're fighting with a mix of ref callbacks, the antipattern of calling methods through refs, class components, `SuggestionBox`, and importing JS code, so I'm not at all surprised that it's causing problems 😅